template/10-timeline.md

```diff
@@ -1,10 +1,92 @@
+
+#### Milestone 2
+
+Post features development
+
+- Develop the core post system that the users will mainly interact with.
+- Ensure that each new feature is thouroghly tested.
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 3 | Post creation | Posts with data can be stored and retrieved from the backend. Frontend elements can display this data. |
+| Week 4 | Commenting system | Completed commenting feature in frontend and backend |
+| Week 5 | Voting system | Developped voting system. Unit testing that ensures that database stays consistent |
+| Week 6 | Location-based feed | Combinations of the developped features in the frontend. Mock testing. |
+
+#### Milestone 3
+
+App screens development
+
+- Develop the different pages of the app 
+- Ensure that each new feature fits well with the rest of the app. 
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 7 | Profile page | Developed profile page that enables the user to manage his personal data |
+| Week 8 | Challenge system | First Gamification of the app. |
+| Week 9 | Map page | Possibility to interact with the different posts and feature trough a map. | 
+| Week 10 | Global app navigation | Ergonomic and functional app.|
+
+#### Milestone 4
+
+Final testing and deployment
+
+- Ensure the stability of the app 
+- Get first feedbacks from testers 
+- Release the app to the public
+
+
+| **Sprint/Week Number** | **Objective** | **Outcomes** |
+| --- | --- | --- |
+| Week 11 | Quality check | Stable version off the app. Tested and validated user stories |
+| Week 12 | Feedback from testers | Pain points, possibilities of ameliorationa and bug identification. |
+| Week 13 | Feedback addressing | Bug fixing, amelioration of the app. Preparation for release on app stores. |
+| Week 14 | First public release | Working app ready for marketing and further updates. |
+
+
+## Development Resources
+
+Human resources represent the main cost of the project. The 2 backend developers will be in charge of managing 
+the database and the API of the backend. The 2 frontend developers will be in charge of the client part of the app. The UI/UX designer will be in charge of the overall design of the app. The project manager will be in charge of the coordination of the team and the communication with the users.
+
+| **Function** | **Required person-months** |
+| --- | --- |
+| Frontend Flutter Developers | 8 |
+| Backend Developers | 8 |
+| UI/UX Designer | 4 |
+| Project Manager | 4 |
+
+## Deployment Resources
+
+We use the services of Google to manage our backend and to monitor the app. The map will be self-hosted with OpenStreetMap.
+
+| **Item** | **Cost / unit** | **Units** | **Totals** |
+| --- | --- | --- | --- |
+| OpenStreetMap Hosting | 200 CHF / month | - | 800 CHF |
+| Google Firebase | 150 CHF / month | - | 600 CHF  |
```

Comment:
```suggestion
| OpenStreetMap Hosting | 200 CHF / month | 4 | 800 CHF |
| Google Firebase | 150 CHF / month | 4 | 600 CHF  |
```

I think you should clarify that we will be using these resources for 4 months here.